### PR TITLE
[LLM] Add exclude option in all-in-one performance test

### PR DESCRIPTION
**Pull request by Mingyu-Wei**
### 1. Why the change?
@hkvision @WeiguangHan 

Add exclude option in all-in-one perf test to avoid running some models that might trigger `RuntimeError: Allocation out of device memory`.

### CODE REVIEW

File: python/llm/test/benchmark/arc-perf-test.yaml [hunk 1]
```diff
@@ -27,3 +27,6 @@ in_out_pairs:
 test_api:
   - "transformer_int4_gpu"  # on Intel GPU
 cpu_embedding: False # whether put embedding to CPU (only avaiable now for gpu win related test_api)
+exclude:
+  - 'baichuan-inc/Baichuan2-13B-Chat:1024'
+  - 'baichuan-inc/Baichuan2-13B-Chat:2048'
```
Comment (hkvision) [inline python/llm/test/benchmark/arc-perf-test.yaml:32]: Baichuan2-13B is not here actually, you should first exclude 1k and 2k of moss?

Comment (Mingyu-Wei) [reply python/llm/test/benchmark/arc-perf-test.yaml]: Sure

File: python/llm/dev/benchmark/all-in-one/run.py [hunk 2]
```diff
@@ -747,6 +757,8 @@ def run_transformer_int4_gpu_win(repo_id,
     from omegaconf import OmegaConf
     conf = OmegaConf.load(f'{current_dir}/config.yaml')
     today = date.today()
+    if 'exclude' in conf:
+        excludes = conf['exclude']
```
Comment (hkvision) [inline python/llm/dev/benchmark/all-in-one/run.py:761]: is this correct? This is in `run_transformer_int4_gpu_win` not in `run_transformer_int4_gpu`?

Comment (hkvision) [inline python/llm/dev/benchmark/all-in-one/run.py:761]: I think we can process exclude here?
Process exclude to be a list of tuples (model, input_len)?
And then should_exclude is not needed, you only need to check if the tuple is in the list?

File: python/llm/dev/benchmark/all-in-one/run.py [hunk 3]
```diff
@@ -747,11 +747,19 @@ def run_transformer_int4_gpu_win(repo_id,
...
     
     import pandas as pd
     for api in conf.test_api:
         for model in conf.repo_id:
-            run_model(model, api, conf['in_out_pairs'], conf['local_model_hub'], conf['warm_up'], conf['num_trials'], conf['num_beams'],
+            in_out_pairs = conf['in_out_pairs'].copy()
+            if 'exclude' in conf:
```
Comment (hkvision) [inline python/llm/dev/benchmark/all-in-one/run.py:757]: can use `if excludes` here?

**Review by hkvision — APPROVED** [review]

File: python/llm/test/benchmark/arc-perf-test.yaml [hunk 4]
```diff
@@ -27,3 +27,6 @@ in_out_pairs:
 test_api:
   - "transformer_int4_gpu"  # on Intel GPU
 cpu_embedding: False # whether put embedding to CPU (only avaiable now for gpu win related test_api)
+exclude:
+  - 'fnlp/moss-moon-003-sft:1024'
+  - 'fnlp/moss-moon-003-sft:2048'
```
Comment (WeiguangHan) [inline python/llm/test/benchmark/arc-perf-test.yaml:32]: Need to add `'bigscience/bloomz-7b1:2048'` which causes OOM error and segment fault randomly.

**Pull request merged**